src/containers/Home/Home.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Button from '../../components/Button/ExampleButton';
+import { addArticle, getArticles } from '../../requests/ArticleRequests';
+
+/**
+ *
+ *
+ * @class Home
+ * @extends {Component}
+ */
+class Home extends Component {
+  /**
+   * @returns {array} article
+   * @memberof Home
+   */
+  componentDidMount() {
+    this.props.getArticles();
```

Comment:
@kampkelly shouldn't this be `this.getAllArticles()` 
you would also need to bind it in the constructor if you are passing this method down as props to a child component